install_script.sh.template

```diff
@@ -237,18 +238,21 @@ flavor_to_system_service=(
     ["datadog-dogstatsd"]="datadog-dogstatsd"
 )
 system_service=${flavor_to_system_service[$agent_flavor]:-datadog-agent}
+system_fips_service=datadog-fips-proxy
```

Comment:
How about directly defining the `services` array here, given that it's only used in one place (when printing the end message)?

```suggestion
declare -a services
services=("${flavor_to_system_service[$agent_flavor]:-datadog-agent}")
if [ -n "$fips_mode" ]; then
  services+=("datadog-fips-proxy")
fi
```